rust/scx_utils/src/cpumask.rs

```diff
@@ -17,36 +17,42 @@
 //! hexadecimal string:
 //!
 //!```
-//!     let all_zeroes = cpumask::new();
-//!     let str = "0xff00ff00";
-//!     let from_str_mask = cpumask::from_string(str);
+//!     use scx_utils::Cpumask;
+//!     let all_zeroes = Cpumask::new();
+//!     let str = String::from("0xff00ff00");
```

Comment:
I think this one is going to fail unless we have all these CPUs inside the vng instance (hence my comment about using just 8 CPUs). So, maybe change this code chunk with:
```
use scx_utils::Cpumask;
let all_zeroes = Cpumask::new();
let str = String::from("0xf0");
```

Reply:
Ahh that makes sense, thank you!